views/relations.py

```diff
@@ -28,6 +28,13 @@ def get_relations(cls, user_id):
         if error_message:
             return json.dumps({"response": error_message}), status
         return json.dumps({"response": list(map(lambda x: x.json() if x else None, response))}), status
+     
+    @classmethod
+    def get_specific_relation(cls, relation_id):
+        error_message, status, response = RelationsController.get_specific_relation(relation_id)
+        if error_message:
+            return json.dumps({"response": error_message}), status
+        return json.dumps({"response": list(map(lambda x: x.individual_json() if x else None, response))}), status
```

Comment:
why are you using map to iterate? There should be only one relation object that's related to one relation_id, as it is a unique field. Also, you don't even use `json()` here, which you created in this pr.

Reply:
heard from Yul Hee, seems like your fix for this hasn't been pushed yet.